### DISCUSSION TITLE
various ring devices

### DISCUSSION THREAD

**Discussion by huesofredy** [tsightler/ring-mqtt#806] [Support Q&A]
I currently have 1 ring device and the idea is to buy another ring Door View Cam. I have docker mqtt and touch from the first token device, how can I get the token from the second device? When I add the account are all devices added?

**Comment by tsightler** [comment]
The token establishes the connection with the Ring API, it is not per-device.  Any device which the account you've logged in to will be discovered automatically.  If you add a new device, once the device is registered in the Ring app, just restart ring-mqtt and it will be discovered.

**Comment by huesofredy** [comment]
Thank you very much, I appreciate your response, I currently have a ring
intercom and the intention is to put a peephole on the main door. I have
seen the Ring door model but you need a subscription, right? Can't detect
and record without subscription. I'm also looking at the ezviz cp4 to
integrate home assistant. Any recommendation? thank you
